icu4c/source/common/locid.cpp

```diff
@@ -102,11 +102,12 @@ typedef enum ELocalePos {
     eMAX_LOCALES
 } ELocalePos;
 
-U_CFUNC int32_t locale_getKeywords(const char *localeID,
-            char prev,
-            char *keywords, int32_t keywordCapacity,
-            UBool valuesToo,
-            UErrorCode *status);
+void U_EXPORT2
+_getKeywords(const char *localeID,
```

Comment:
Please keep the "U_CFUNC" for a function that is shared across .cpp files. I don't remember if it *also* needs U_EXPORT2 -- looks like it worked without that.

https://unicode-org.github.io/icu/userguide/dev/codingguidelines.html#function-declarations

Reply:
Done.